### ISSUE TITLE
UTSUtil.isTrackingEnabled NullPointerException

### ISSUE THREAD

**Issue by akndmr** [zoho/salesiq-mobilisten-android-sample#8]
After Zoho SalesIq update to v2.2, we started to get below error with some users. Do you have any guess about the reason?
Thanks!

```
Caused by java.lang.NullPointerException
Attempt to invoke virtual method 'boolean java.util.ArrayList.contains(java.lang.Object)' on a null object reference

com.zoho.salesiqembed.android.tracking.UTSUtil.isTrackingEnabled (UTSUtil.java:576)
com.zoho.salesiqembed.ZohoSalesIQ$1.onActivityResumed (ZohoSalesIQ.java:240)
android.app.Application.dispatchActivityResumed (Application.java:455)
androidx.fragment.app.FragmentActivity.onResume (FragmentActivity.java:455)
com.xxx.BaseActivity.onResume (BaseActivity.java:95)
```

**Comment by Angu-G** [comment]
We will fix this in our next immediate update. It can be expected in a couple of weeks.

**Comment by Angu-G** [comment]
@akndmr We have a fix for this in our latest update (2.3.1).

**dependency:**
implementation 'com.zoho.salesiq:mobilisten:2.3.1'
